projects/gen-ai.md

```diff
@@ -0,0 +1,164 @@
+- discuss implementation challenges
+- coordinate and prioritize common efforts
+
+### Goals, objectives, and requirements
+
+GenAI observability project is a new OTel cross-cutting SIG that intends to provide the GenAI community with a space to evolve and maintain GenAI related instrumentations in a **vendor-agnostic way**.
+
+It includes:
+
+- GenAI Semantic Conventions
+- Instrumentations for GenAI client libraries, frameworks, or other components.
+
+The eventual goal is to contribute instrumentation libraries to upstream components or to otel-contrib repos when upstream contributions are not possible.
+
+During initial development and experimentation, we may need to release libraries more frequently than it's possible for contrib and then may temporarily host instrumentation libraries in a `opentelemetry-{language}-genai` repository(ies).
+
+This would help GenAI observability vendors that will depend on these libraries to patch issues and adapt to external changes.
+
+## Deliverables
+
+Immediate term:
+- Ship OTel instrumentation libraries for OpenAI (or any other GenAI client) in Python and JS following existing conventions
+
+Middle term:
+- Ship OpenTelemetry (or native) instrumentations for popular GenAI client libraries in Python and JS covering chat calls
+- Evolve GenAI semantic conventions to cover other popular GenAI operations such as embeddings, image or audio generation
+
+As a result we should have feature parity with the instrumentations of existing GenAI Observability vendors for a set of client
+instrumentation libraries that all vendors can depend upon.
+
+Long term:
+- Implement instrumentations for GenAI orchestrators and GenAI frameworks for popular libraries in different languages
+- Evolve GenAI and other relevant conventions (DB) to cover complex multi-step scenarios such as RAG
+- Propose mature instrumentations to upstream libraries/frameworks
+
+## Staffing / Help Wanted
+
+The initial set of approvers for each language must include:
+
+- representatives of at least 2 different observability vendors that are active members of OTel GenAI SemConv WG
+- at least one existing approver or maintainer of the contrib, instrumentation or core OTel repo in the corresponding language
+
+**Project Leads:**
+
+- @drewby
+- @nirga
+
+**Sponsoring Members:**
+
+- @lmolkova
+- TBD
+
+**Engineers:**
+
+- **initial** maintainers (based on the existing membership in the corresponding repo)
+  - Python
+    - @lzchen
+    - @xrmx
+    - TBD
+  - JavaScript
+    - @trentm
+    - *looking for existing OTel JS maintainers for general JS & OTel feedback*
+    - TBD
+- **initial** JS and Python GenAI component/code owners (based on LLM Semantic Convention project involvement):
+  - @drewby
+  - @karthikscale3
+  - @nirga
+- contributors:
+  - @AmanAgarwal041 (JS & Python)
+  - @alizenhom (JS & Python)
+  - @codefromthecrypt (JS & Python)
+  - @galkleinman (JS & Python)
+  - @gyliu513 (JS & Python)
```

Comment:
@lmolkova I want to nominate myself to join OTEL community as a member, but I only did some PR reviews and attend some LLM Semantic Convention meetings, what do you think of counting in the effort that we contribute to https://github.com/traceloop/openllmetry/tree/main/packages as well? All of the instrumentation code are based on OpenTelemetry, and I think this may become the code base for the python instrumentation code, it will be great if we can count in those efforts for someone to apply to be a member of otel community, what do you think? Thanks!

Reply:
sorry I missed your comment - I believe you are now member of the community :) welcome and thanks for your contributions!

Reply:
Yes, thank you for sponsoring my membership @lmolkova :-)